Library/Formula/libfabric.rb

```diff
@@ -0,0 +1,18 @@
+class Libfabric < Formula
+  desc "OpenFabrics libfabric"
+  homepage "https://ofiwg.github.io/libfabric/"
+  url "http://downloads.openfabrics.org/downloads/ofi/libfabric-1.1.1.tar.bz2"
+  sha256 "228c50d1f9595009a026bb2293f372a04d89ecb3b0f7fcde3905476dbf49fc95"
+  head "https://github.com/ofiwg/libfabric"
```

Comment:
This should have `.git` at the end: `head "https://github.com/ofiwg/libfabric.git"`. Sorry I didn't catch that the first time around.